nima/webclient/webclient/src/main/java/io/helidon/nima/webclient/http1/ClientRequestImpl.java

```diff
@@ -356,6 +317,75 @@ ClientConnection getConnection(boolean keepAlive) {
         return connection;
     }
 
+    // See https://www.rfc-editor.org/rfc/rfc7230#section-3.2
+    private static void validateHeaderValues(HeaderValue header) throws IllegalArgumentException {
```

Comment:
I think that this method may be better suited to `HeaderValue` directly, as it represents an HTTP Header value, and the same rules apply to any version of HTTP I think.
Then we could use the same validation logic regardless of server/client